modules/apache-airflow/templates/values.yaml

```diff
@@ -2354,7 +2354,7 @@ redis:
 # Auth secret for a private registry
 # This is used if pulling airflow images from a private registry
 registry:
-  secretName: ~
+  secretName: docker-cfg
```

Comment:
How would it work if we ever were to pull an image from another registry such as quay.io/would we ever need to do that?